From aa4d00ca39e604ac4e9fead401ccd4483e11a281 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 25 Nov 2016 11:12:10 -0500
Subject: [PATCH] localedata: GBK: add mapping for 0x80->Euro sign [BZ #20864]

Microsoft long ago added a mapping for 0x80 to the Euro sign to their
CP936.  While GBK 1.0 doesn't include this mapping, it is compatible,
and Microsoft and glibc alias the two codepages.  We could split them
apart so GBK wouldn't include the mapping, but that seems like a lot
of work for little gain.
---
 localedata/charmaps/GBK |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/localedata/charmaps/GBK b/localedata/charmaps/GBK
index 4c35f14..4186896 100644
--- a/localedata/charmaps/GBK
+++ b/localedata/charmaps/GBK
@@ -3,10 +3,15 @@
 <mb_cur_min> 1
 <comment_char> %
 <escape_char> /
+% https://en.wikipedia.org/wiki/GBK
 
+% https://www-01.ibm.com/software/globalization/ccsid/ccsid936.html
 % alias CP936
+
+% ftp://ftp.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit
 % alias MS936
 % alias WINDOWS-936
+
 CHARMAP
 <U0000>     /x00         NULL (NUL)
 <U0001>     /x01         START OF HEADING (SOH)
@@ -136,6 +141,8 @@ CHARMAP
 <U007D>     /x7d         RIGHT CURLY BRACKET
 <U007E>     /x7e         TILDE
 <U007F>     /x7f         DELETE (DEL)
+% This isn't in GBK 1.0, but Microsoft added it to their CP936 page.
+<U20AC>     /x80         EURO SIGN
 
 <U4E02>     /x81/x40     <CJK>
 <U4E04>     /x81/x41     <CJK>
-- 
1.7.1

